data_collection/gazette/spiders/sp_araraquara.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+from urllib.parse import urlencode
+
+import dateparser
```

Comment:
**issue** Imported but unused ([F401](https://www.flake8rules.com/rules/F401.html)). Remove this import.